tendrl/ceph_integration/objects/tendrl_context/__init__.py

```diff
@@ -83,10 +83,10 @@ def _get_sds_details(self):
             return None
 
         if out:
-            details = json.loads(out)
+            details = out.split()
```

Comment:
Please add comment mentions the output of the ceph --version, which helps th ereview